department-of-veterans-affairs/va.gov-cms
# VACMS-7521 vagovclaro css regressions

## Description

Relates to #7521. Builds on work in #7639 to ensure styles are consistent with the patch applied.

- Moves paragraph field help text back into header
- Makes subform buttons for adding paragraphs & IEF actions small, helps spacing
- Clean up some spacing around paragraph buttons, rich texts
- Ensures centralized content editors see the correct styles, fields allowed to edit
- Gives centralized content scopes a colored background on the view tab again
- Fixes proofing styles to correctly apply. Node view page looks much better & consistent to current theme. 


## Testing done
- admin user, view centralized content, vamc system, vet centers
- log in as randi.hecht@va.gov check centralized content, vamc system, vet centers
- verified changes against control tugboat environment that is using the old theme to ensure consistency

## Screenshots
![Screen Shot 2022-01-19 at 9 41 18 AM](https://user-images.githubusercontent.com/11279744/150202888-3bb4ed19-5c4f-4348-a5a3-2cdd9aa417a0.png)
![Screen Shot 2022-01-19 at 10 11 17 AM](https://user-images.githubusercontent.com/11279744/150202891-998bc2bf-c547-4652-ae3a-886449e9f79c.png)
![Screen Shot 2022-01-19 at 10 11 23 AM](https://user-images.githubusercontent.com/11279744/150202892-d6d565eb-e9ab-401d-8d82-41e489bfa575.png)
![Screen Shot 2022-01-19 at 11 42 05 AM](https://user-images.githubusercontent.com/11279744/150202895-4a3a4733-6d2d-4a6a-9480-f081bb254058.png)
![Screen Shot 2022-01-19 at 11 42 48 AM](https://user-images.githubusercontent.com/11279744/150202896-a97338ab-fec3-4987-8b73-93cd04cb7558.png)
![Screen Shot 2022-01-19 at 11 42 53 AM](https://user-images.githubusercontent.com/11279744/150202898-08384517-9cbc-4dac-9514-4acd3598a285.png)
![Screen Shot 2022-01-19 at 11 45 22 AM](https://user-images.githubusercontent.com/11279744/150202902-be82d827-e5cd-442f-ac83-a388c5006c42.png)
![Screen Shot 2022-01-19 at 11 45 28 AM](https://user-images.githubusercontent.com/11279744/150202905-8f603bab-7b37-46e7-b647-3be99651acbe.png)


## QA steps
as admin and lower user, check these nodes and see that the editor experience has improved:
- `/node/905/edit` & `/disability/how-to-file-claim` [view tab]
- `/node/33340/edit` & `/centralized-content/vamc-system-medical-records-page-content` [view tab]
- `/node/3877/edit`

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
